### PR TITLE
[CCXDEV-8782] Service log initial steps

### PR DESCRIPTION
**Pull request by juandspy**
# Description

Two basic tests for service log.

## Type of change

- New test feature file
- New test scenario added into existing feature file

## Testing steps

Locally.

## Checklist
* [ ] Pylint passes for Python sources
* [ ] sources has been pre-processed by Black
* [ ] updated documentation wherever necessary

### CODE REVIEW

File: features/ccx-notification-service/customer_notifications.feature [hunk 1]
```diff
@@ -1,35 +1,49 @@
...
 
+  Background: Kafka is empty
+    Given Kafka topic "platform.notifications.ingress" is empty
+      And CCX Notification database is created for user postgres with password postgres
+      And CCX Notification database is empty 
+      And insights-content service is available on localhost:8082
 
   Scenario: Check that notification service does not need kafka if database has no new report
     Given Postgres is running
-      And CCX Notification database is created for user postgres with password postgres
```
Comment (tisnik) [inline features/ccx-notification-service/customer_notifications.feature:6]: does it work w/o DB at all?

File: features/ccx-notification-service/service_log.feature [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      And service-log service is empty
+     When I insert 1 report with important total risk for the following clusters
+          | org id |  account number | cluster name                         |
+          | 1      |  1              | 5d5892d4-2g85-4ccf-02bg-548dfc9767aa |
+      And I start the CCX Notification Service with the --instant-reports command line flag
+          | val                                             | var   |
+          | CCX_NOTIFICATION_SERVICE__SERVICE_LOG__ENABLED  | false |
+     Then it should have sent 0 notification events to Service Log
+      And the process should exit with status code set to 0
+      And The logs should match
```
Comment (tisnik) [inline features/ccx-notification-service/service_log.feature:20]: pro lowercase `the` (nitpick)

File: features/steps/kafka_util.py [hunk 3]
```diff
@@ -66,3 +66,39 @@ def find_available_brokers(context):
     assert (
         len(context.broker_metadata["brokers"]) >= 1
     ), "At least one available broker expected"
+
+
+@given('Kafka topic "{topic}" is empty')
+def make_kafka_empty(context, topic):
+    """Delete all records from Kafka."""
```
Comment (tisnik) [inline features/steps/kafka_util.py:73]: records/events

**Review by tisnik — APPROVED** [review]
LGTM

File: features/ccx-notification-service/customer_notifications.feature (hunk 1 above)
Comment (juandspy) [reply features/ccx-notification-service/customer_notifications.feature:6]: this step have been moved to the `Bakground`, so the database is created in each step. It was just a way of not duplicating the same line in every scenario

Comment (tisnik) [reply features/ccx-notification-service/customer_notifications.feature:6]: yes sure, I meant if we have a test for "problematic DB" cases

Comment (juandspy) [reply features/ccx-notification-service/customer_notifications.feature:6]: ah, don' think so :/

**Pull request merged**